### PR TITLE
Readme: drop redundant composer info

### PR DESCRIPTION
**Pull request by TomasVotruba**
These lines now can be removed, since composer is standard way.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -12,17 +12,12 @@ but can now be used stand-alone.
 
 ## Installation
 
-KnpMenu uses Composer, please checkout the [composer website](http://getcomposer.org) for more information.
-
-The simple following command will install `knp-menu` into your project. It also add a new
-entry in your `composer.json` and update the `composer.lock` as well.
+Install with Composer (please checkout the [composer website](http://getcomposer.org) for more information):
```
Comment (stof) [inline README.markdown]: please switch the composer link to HTTPS (it would redirect to it anyway)

**Comment by Nek-** [comment]
I don't see the benefit of removing information...

**Comment by TomasVotruba** [comment]
While providing composer install info, this information is irrelevant, since it's managed by composer internally. 

It would make sense if there would be described other ways to install it.

What was the benefit of adding it?

**Comment by dbu** [comment]
i am +1 on this one. only tell relevant things to the users, makes it easier to grasp the important things. if people don't know about composer and its autoloading, they should read about it in a tutorial for that topic. its not enough for the total noob, and everybody else does not need the information as its obvious with composer.

**Comment by TomasVotruba** [comment]
@dbu That exactly my thought process.

**Comment by WouterJ** [comment]
Can you please update this PR to use the [standardized bundle installation template](http://symfony.com/doc/current/cookbook/bundles/best_practices.html#documentation)?

**Comment by TomasVotruba** [comment]
Seems like not ready yet, closing.

**Pull request closed without merging**